util/humantime/humantime.go

```diff
@@ -1,13 +1,17 @@
-package sshchat
+package humantime
```

Comment:
Maybe this should be internal/humantime instead of util/humantime?

Also personally I call these types of packages `pretty`, like `pretty.Since(...)`. But humantime is fine too. :)